internal/runners/packages/packages.go

```diff
@@ -105,7 +104,7 @@ func executePackageOperation(prime primeable, packageName, packageVersion string
 
 	// Verify that the provided package actually exists (the vcs API doesn't care)
 	_, err = model.FetchRecipe(commitID, pj.Owner(), pj.Name(), &model.HostPlatform)
-	if err != nil && !inventory.IsPlatformError(err) {
+	if err != nil && !model.IsPlatformError(err) {
```

Comment:
I am a little confused that we are ignoring platform errors here.  That basically means that we allow to add anything to the order, if the resulting solver error is that this package is not supported by the platform.

Reply:
It's ignoring platform errors as in the platform of the build (ie. the OS). Not platform as in "ActiveState Platform".

Reply:
> It's ignoring platform errors as in the platform of the build (ie. the OS). Not platform as in "ActiveState Platform".

I understand.  So, the idea is that we can add all kinds of ingredients to a MacOS platform, even if we do not know how to build it.  But I am still surprised, and wonder if this is really what we want to do here.